### ISSUE TITLE
button_type is checking containment in two different sets

### ISSUE THREAD

**Issue by jsignell** [pyviz/panel#274]
I can get both these error messages when using the button_type args:

```
ValueError: warning not in Parameter button_type's list of possible objects, valid options include [default, primary, success, info, danger]

ValueError: invalid value: 'info'; allowed values are default, primary, success, warning or danger
```

**Comment by philippjfr** [comment]
Not sure where I got 'info' from, seems like I confused it with 'warning'.

**Comment by philippjfr** [comment]
Fixed in #283
